front/app/components/UI/ClaveUnicaButton/index.tsx

```diff
@@ -56,42 +54,56 @@ const ButtonIcon = styled.span`
   box-sizing: border-box;
 `;
 
-const ButtonLabel = styled.span`
-  padding-left: 3px;theme.
+const ClaveUnicaButtonLabel = styled.span`
+  padding-left: 3px;
   text-decoration: underline;
   box-sizing: border-box;
 `;
 
 const HelperText = styled.div`
+  display: inline;
+  margin-left: 10px;
   color: ${(props) => props.theme.colors.tenantText};
 `;
 
 interface Props {
   method: TVerificationMethod;
   last: boolean;
   onClick: (method: TVerificationMethod) => void;
+  message: string | JSX.Element;
+  disabled?: boolean;
 }
 
-const BosaFasButton = ({ method, last, onClick }: Props) => {
+const ClaveUnicaButton = ({
+  method,
+  onClick,
+  message,
+  disabled = false,
+}: Props) => {
   const handleOnClick = () => {
     onClick(method);
+    // Probably, it doesn't affect the functionality. The actual location change happens in
+    // setHref of front/app/services/singleSignOn.ts
+    // TODO: remove.
     const jwt = getJwt();
     window.location.href = `${AUTH_PATH}/clave_unica?token=${jwt}&pathname=${removeUrlLocale(
       window.location.pathname
     )}`;
   };
 
   return (
-    <Container last={last}>
-      <ButtonWrapper onClick={handleOnClick} id="e2e-clave_unica-button">
-        <ButtonIcon />
-        <ButtonLabel>Iniciar sesión</ButtonLabel>
+    <div>
+      <ButtonWrapper
+        onClick={disabled ? undefined : handleOnClick}
+        id="e2e-clave_unica-button"
+        className={disabled ? 'disabled' : undefined}
+      >
+        <ClaveUnicaButtonIcon />
+        <ClaveUnicaButtonLabel>Iniciar sesión</ClaveUnicaButtonLabel>
```

Comment:
Should "Iniciar sesión" be coming from a translated message?

Reply:
It was like that before. But good point, let me change it.

Reply:
I've just realized that ClaveUnica guideline says it should be exactly this text https://drive.google.com/file/d/1-aBGu5XEjHD1LYcqOZP_mUg4ekFLDv6v/view
So, probably it should not be translated.